app/views/layouts/application_form_mailer.slim

```diff
@@ -0,0 +1,63 @@
+doctype transitional
+html[lang="en"]
+  head
+    meta[http-equiv="Content-Type" content="text/html; charset=UTF-8"]
+    meta[name="viewport" content="width=device-width, initial-scale=1"]
+    meta[http-equiv="X-UA-Compatible" content="IE=edge"]
+    title
+    style[type="text/css"]
+
+  body[style="margin:0; padding:0; background-color:#F2F2F2;"]
+    center
+      table[width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#F2F2F2"]
+        tr
+          td[align="center" valign="top"]
+            / start header
+            table[width='640' cellpadding='0' cellspacing='0' border='0' bgcolor='#d82222']
+              tr
+                td[height='20' style='font-size:10px; line-height:10px;']
+                  |
+              tr
+                td[align='center' valign='top']
+                  table[width='600' cellpadding='0' cellspacing='0' border='0']
+                    tr
+                      td[align='left' valign='top']
+                        = image_tag('logo.png', width: 150)
```

Comment:
I cannot remember the last time I dealt with images in mails send *directly* from a Rails app... I'm not 100% sure about the use of the `image_tag` helper there. Is it a save thing to do? Or do I need to do a "normal" `<img>` tag and then point the source to the asset **url**?

Reply:
I think this is fine, but don't have a lot of experience in using images in emails